src/test/kotlin/no/nav/familie/ef/sak/beregning/BeregningServiceTest.kt

```diff
@@ -416,4 +417,20 @@ internal class BeregningServiceTest {
             )
         }
     }
+
+    @Test
+    fun `grunnbeløpsperiodeDTO skal returnere korrekt seks ganger grunnbeløp per måned`() {
+        val g = 124028.toBigDecimal()
+        val grunnbeløp =
+            Grunnbeløp(
+                periode = Månedsperiode(LocalDate.of(2024, 5, 1), LocalDate.of(2025, 4, 30)),
+                grunnbeløp = g,
+                perMnd = 10335.toBigDecimal(), // G 124028/12 = 10335.6666667
```

Comment:
Skal vi runde ned her?
Trodde `half_up` skulle ha rundet dette opp til `10336`?

Ift denne så ser det ut som 10336 brukes også - https://lovdata.no/nav/rundskriv/v6-03-00

Reply:
Den er satt til 10335 her:

```
val grunnbeløpsperioder: List<Grunnbeløp> =
    listOf(
        Grunnbeløp(
            periode = Månedsperiode(YearMonth.parse("2024-05"), YearMonth.from(LocalDate.MAX)),
            grunnbeløp = 124_028.toBigDecimal(),
            perMnd = 10_335.toBigDecimal(),
            gjennomsnittPerÅr = 122_225.toBigDecimal(),
        ),
        ...
```

Hvis vi endrer perMnd over, er det ikke behov for å gjøre disse endringene på utregningen 👍

Reply:
Da kanskje vi kan endre direkte istedenfor? Vi var litt usikre på hva som er "riktig tall" - og sett i ettertid brukte vi nok feil tall der. 

Men bør uansett endre utregningen, for `seksGangerGrunnbeløpPerMåned = 6.toBigDecimal() * grunnbeløpMåned` blir jo upresist med både `10_335` og `10_336`?

Reply:
Ja, ser nå at gammel utregning med `10_336` også blir feil. Stemmer for å beholde PR som den er nå.